test/testdata/bootstrappolicy/bootstrap_cluster_role_bindings.yaml

```diff
@@ -812,7 +812,7 @@ items:
     annotations:
       rbac.authorization.kubernetes.io/autoupdate: "true"
     creationTimestamp: null
-    name: admin
+    name: template-instance-controller:admin
```

Comment:
`system:openshift:controller:template-instance-controller:admin`  "don't muck with this: it's for openshift: for a controller: this controller: bound to that role"